source/manual/deploying.html.md

```diff
@@ -27,12 +27,13 @@ We have a staging environment that must always be deployed to immediately before
 ## Release process
 
 Only one release takes place at a time. One product team owns the release - if
-multiple teams are involved in the release, pick one.
+multiple teams are involved in the release, pick one. Releases are tracked by the [release app](https://release.publishing.service.gov.uk/).
 
+You need to have a Signon account with appropriate permissions to access the release app.
 
-## Experimental deployment process
+## Deployment process
 
-As a response to [RFC-70](https://gov-uk.atlassian.net/wiki/pages/viewpage.action?pageId=130703581) starting 15 May 2017 we are experimenting with a process which allows us to remove the need to book a deployment slot. If you think there's something wrong with the process during the experiment speak up as soon as possible.
+As a response to [RFC-70](https://gov-uk.atlassian.net/wiki/pages/viewpage.action?pageId=130703581) starting 15 May 2017 we are using a process which allows us to deploy without the need for a booked deployment slot.
```

Comment:
The link to the RFC should point to the [RFC on GitHub](https://github.com/alphagov/govuk-rfcs/blob/master/rfc-070-path-towards-continuous-deployment-cd.md) now.